src/mon/PGMap.cc

```diff
@@ -1970,6 +2039,24 @@ void PGMap::dump_stuck_plain(ostream& ss, int types, utime_t cutoff) const
     dump_pg_stats_plain(ss, stuck_pg_stats, true);
 }
 
+int PGMap::dump_stuck_unavailable_pg_stats(
+  //TODO: make it such that we have poolid as key
+  // and value to be: pgid, state and etc.
+  ceph::Formatter *f,
+  int threshold) const
+{
+  int stuck_types = 0;
+  stuck_types |= PGMap::STUCK_INACTIVE;
+  stuck_types |= PGMap::STUCK_STALE;
+  stuck_types |= PGMap::STUCK_PEERING;
```

Comment:
@ljflores here but instead of creating a new function, just add the ``stuck_types |= PGMap::STUCK_PEERING;`` to the already existing function: ``dump_stuck_pg_stats``

Reply:
@shreyanshjain7174